radio/src/lua/api_general.cpp

```diff
@@ -1068,7 +1068,7 @@ static int luaSetTelemetryValue(lua_State * L)
     zname[2] = hex2zchar((id & 0x00f0) >> 4);
     zname[3] = hex2zchar((id & 0x000f) >> 0);
   }
-  
+
   int index = setTelemetryValue(TELEM_PROTO_LUA, id, subId, instance, value, unit, prec);
```

Comment:
We need another check before the call of `setTelemetryValue()` the `id`, `subId` and `instance` must not be zero - this would then use the first empty sensor slot without actually allocating it as a new sensor!